pfioh/pfioh.py

```diff
@@ -214,7 +214,10 @@ def do_GET_withCompression(self, d_msg):
                                 cleanup     = b_cleanup, 
                                 d_ret       = d_ret
                             )
-        
+        if d_ret['status']:
```

Comment:
Since the SwiftStore class no longer exists, the logic for returning the data is now in pfioh.py so I can use self.wfile.